src/sbt-test/sbt-scalafix/basic/build.sbt

```diff
@@ -25,3 +25,19 @@ lazy val example = project
   )
 
 lazy val tests = project
+
+lazy val checkLogs = taskKey[Unit]("Check that diffs are logged as errors")
+
+checkLogs := {
+  val taskStreams = (example / Compile / scalafix / streams).value
+  val reader = taskStreams.readText(taskStreams.key)
```

Comment:
The `lastLogFile` used in a [few scripted](https://github.com/sbt/sbt/blob/4c3cd4e535a89d220a0d20f360f36212ceff9b09/sbt/src/sbt-test/reporter/nowarn/build.sbt#L10-L11) did not work as it seems that the except failure just before this gets called flush out the output... In the end, this looks more readable IMHO.